targets/esp8266/docs/ESP-PREREQUISITES.md

```diff
@@ -56,14 +58,9 @@ PATH=$PWD/builds/xtensa-lx106-elf/bin:$PATH
 cd /opt/Esprissif
 git clone https://github.com/espressif/ESP8266_RTOS_SDK.git ESP8266_RTOS_SDK.git
 ln -s ESP8266_RTOS_SDK.git ESP8266_SDK
-cd ESP8266_SDK
-git checkout -b jerry a2b413ad2996450fe2f173b6afab243f6e1249aa
 ```
 
-We use SDK 1.2.0 version which has stdlib.h and others. Latest 1.3.0 version,
-as of writing this document, doesn't have it.
-(If anyone knows how to use latest version, please add an issue or send a PR.)
-
+We use latest SDK master.
```

Comment:
Is this really a good idea? I'd prefer a fix version.

Reply:
Using the latest master requires less maintenance than if we use a fix version. Also if you check https://github.com/espressif/ESP8266_RTOS_SDK/commits/master you can see improvements and fixes are common. That's why I'd prefer the latest master but @akosthekiss what's opinion about it?

Reply:
> Using the latest master requires less maintenance than if we use a fix version.

This is not true at all.

Reply:
I agree with @LaszloLango . That another repository can add not only fixes and improvements but incompatible changes, too. And then, all of a sudden, this code will not be working/compiling anymore.

In general, when working with external repos, it's good practice to state the last known working revision. Whether it is "formalized" with git submodules or simply documented in a readme/doc (like here should be) is secondary. The key point is that the maintainer should actively update that reference when it's been confirmed that the new version/revision works -- not reactively, when it turns out that something got broken.

Reply:
Seems true. I'll update the documentation.